lib/ansible/playbook/task_include.py

```diff
@@ -69,6 +69,8 @@ def load(data, block=None, role=None, task_include=None, variable_manager=None,
             raise AnsibleParserError('Invalid options for %s: %s' % (task.action, ','.join(list(bad_opts))), obj=data)
 
         if not task.args.get('_raw_params'):
+            if task.action == 'import_tasks':
```

Comment:
I'd remove this comparison altogether, and after this just check if `_raw_params` is still empty.